apps/app-backend/api/internal.yaml

```diff
@@ -435,6 +435,9 @@ components:
               required:
                 - id
                 - name
+            group_id:
+              type: string
+              minLength: 1
```

Comment:
**chore**: we actually don't need to propagate the incoming new `group_id` property to "App IO services storage".

`group_id` property will be removed before synching `app-backend` (db) from `db-services-cms`
```suggestion
            group_id:
              type: string
              minLength: 1
```

Reply:
removed unintended changes in 51cbbcea7d9b624488367d4790a68d778192874e